### PR TITLE
Fix linux warnings Wformat

### PR DESCRIPTION
**Pull request by DmEye**

### CODE REVIEW

**Comment by AlexPeshkoff** [comment]
If you anyway touch this code - may be better remove cast to SLONG?

File: src/include/fb_types.h [hunk 1]
```diff
@@ -41,11 +41,13 @@
 	typedef unsigned int ULONG;
 	const SLONG SLONG_MIN = INT_MIN;
 	const SLONG SLONG_MAX = INT_MAX;
+	#define SLONG_STR_FORMAT "%d"
 #elif SIZEOF_LONG == 4
 	typedef long SLONG;
 	typedef unsigned long ULONG;
 	const SLONG SLONG_MIN = LONG_MIN;
 	const SLONG SLONG_MAX = LONG_MAX;
+	#define SLONG_STR_FORMAT "%ld"
```
Comment (dyemanov) [inline src/include/fb_types.h:50]: We already have `SLONGFORMAT` defined in `common.h`, why yet another define?

**Comment by DmEye** [comment]
> If you anyway touch this code - may be better remove cast to SLONG?

Perhaps it may be better. But the cause of warning was the usage of wrong format. So, I assumed the fix is usage of proper format.

File: src/include/fb_types.h (hunk 1 above)
Comment (DmEye) [reply src/include/fb_types.h:50]: Thanks for the note. I didn't know.

**Review by dyemanov — APPROVED** [review]

**Pull request merged**